src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -31,19 +31,14 @@
 
 import javax.annotation.Nullable;
 
+import com.ning.http.client.*;
```

Comment:
You should configure IntelliJ to follow the Kill Bill code style: https://github.com/killbill/killbill/tree/master/.idea.

Specifically: https://github.com/killbill/killbill/blob/master/.idea/codeStyleSettings.xml#L9-L10